doc/Publishing.md

```diff
@@ -87,6 +87,24 @@ For example, if you picked `0.1.0` as a version then you should run:
 
 To confirm that the tagging was correctly done, check the repo's [releases](https://github.com/eclipse-theia/theia/releases) page and confirm the release just done is listed there.
 
+## Updating the release page on eclipse
+- Login to eclipse: https://projects.eclipse.org/projects/ecd.theia
```

Comment:
Should it be 'Eclipse Foundation' and not Eclipse?

Reply:
Done on next commit

Reply:
maybe even something like: "Navigate to our Eclipse Foundation projrct page and logon"

Reply:
Please address this comment